dali/operators/reader/coco_reader_op.cc

```diff
@@ -39,6 +39,17 @@ Tensor (`m` * `[x, y, w, h] or `m` * [left, top, right, bottom]`) and labels as
   .AddOptionalArg("ltrb",
       R"code(If true, bboxes are returned as [left, top, right, bottom], else [x, y, width, height].)code",
       false)
+  .AddOptionalArg("masks",
+      R"code(If true, segmentation masks are read and returned as polygons.
+Each mask can be one or more polygons. A polygon is a list of points (2 floats).
+For a given sample, the polygons are represented by two tensors:
+  `masks_meta` -> list of tuples (mask_idx, start_idx, count)
+  `masks_coords`-> list of (x,y) coordinates
+One mask can have one or more `masks_meta` having the same `mask_idx`, which means that the mask for that given
+index consists of several polygons).
+`start_idx` indicates the index of the first coords in `masks_coords`.
+Skip `is_crowd=1` annotations.)code",
```

Comment:
?

Reply:
It refers to the COCO dataset format.
```
"iscrowd": 0 if your segmentation based on polygon (object instance)
"iscrowd": 1 if your segmentation based uncompressed RLE (crowd)
```
I'll expand upon the docstring and change it to `iscrowd`

Reply:
The more verbose description is the thing I'm asking for.

Reply:
Done